openmdao/api.py

```diff
@@ -66,7 +66,7 @@
     from openmdao.drivers.pyoptsparse_driver import pyOptSparseDriver
 except ImportError:
     pass
-from openmdao.drivers.scipy_optimizer import ScipyOptimizer
+from openmdao.drivers.scipy_optimizer import ScipyOptimizer, ScipyOptimizerDriver
```

Comment:
a small difference maybe, but the suggestion was `ScipyOptimizeDriver`, as in a driver based on `scipy.optimize`  (nor `r` on Optimize)  Hey it saves you a keystroke! ;)